src/views/Signup/Signup.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import DevC from '../../asset/imgs/devc.png';
+import DevC1 from '../../asset/imgs/devc1.png';
+import DevC2 from '../../asset/imgs/devc2.png';
+import DevC3 from '../../asset/imgs/devc3.png';
+import UserNameIcon from '../../asset/icons/username.png';
+import PasswordIcon from '../../asset/icons/password.png';
+
+
+class Signup extends Component {
+    render() {
+        return (
+            <div className="signup">
+                <div id="carouselExampleIndicators" className="carousel slide" data-ride="carousel">
+                    <div className="carousel-inner">
+                        <div className="carousel-item active">
+                            <img className="d-block w-100" src={DevC} alt="First slide" />
+                        </div>
+                        <div className="carousel-item">
+                            <img className="d-block w-100" src={DevC1} alt="Second slide" />
+                        </div>
+                        <div className="carousel-item">
+                            <img className="d-block w-100" src={DevC2} alt="Third slide" />
+                        </div>
+                        <div className="carousel-item">
+                            <img className="d-block w-100" src={DevC3} alt="Forth slide" />
+                        </div>
+                        <div className="carousel-item">
+                            <img className="d-block w-100" src={DevC} alt="Fifth slide" />
+                        </div>
+                        <div className="carousel-item">
+                            <img className="d-block w-100" src={DevC} alt="Sixth slide" />
```

Comment:
Same here also (update `alt` description e.g `alt=" student holing a book, used as the sixth slide"`)